src/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { Comment } from '../types/Comment';
+import { User } from '../types/User';
+import { Post } from '../types/Post';
+
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type PreparedPosts = {
+  user: User | null,
+  comments: Comment[],
+  post: Post
+};
+
+type Props = {
+  posts: PreparedPosts[]
+};
+
+export const PostInfo:React.FC<Props> = (props) => {
+  const { posts } = props;
+
+  return (
+    <>
+      {posts.map(postElement => {
+        const { post, user, comments } = postElement;
+        const { body, title, id } = post;
+
+        const userComment = comments.filter(comment => comment.postId === post.id);
+
+        return (
+          <ul className="list" key={id}>
+            <li className="list__title">
+              {title}
+            </li>
+            <li className="list__body">
+              {body}
+            </li>
+            <UserInfo userProps={user} />
+            <CommentList commentsProps={userComment} />
```

Comment:
```suggestion
            <li> 
              <p className="list__title">
                {title}
              </p>
              <p className="list__body">
                {body}
              </p>
              <UserInfo userProps={user} />
              <CommentList commentsProps={userComment} />
            </li>
```
better to use one `li` with content inside